ddp-workspace/projects/ddp-sdk/src/lib/services/analyticsEvents.service.ts

```diff
@@ -1,45 +1,46 @@
 import { Injectable } from '@angular/core';
-
-declare let ga: Function;
+import { Subject, Observable } from 'rxjs';
+import { AnalyticsEvent } from '../models/analyticsEvent';
 
 @Injectable()
-export class GoogleAnalyticsEventsService {
+export class AnalyticsEventsService {
+  private events = new Subject<AnalyticsEvent>();
+
   public emitCustomEvent(
     eventCategory: string,
     eventAction: string,
```

Comment:
Noticing that in the great majority of the calls to this method we specify for both the `eventCategory` argument and the `eventAction` argument an enum value from `AnalyticsActionTypes`  suggesting that semantically something is probably incorrect here. Either the enums are not really "action types" or the argument names are misleading. Could you take a look at Google Analytics API to see how these argument align with what gets reported back to Google Analytics?

Reply:
Function's arguments match Google Analytics API, okay, I will refactor enum to make it more semantically correct.

Reply:
Same comment with this one: this is an observation, not a direction to change. If you can come up with a simple solution to refactor to make it cleaner awesome.

Reply:
Done :)